db.go

```diff
@@ -322,3 +322,49 @@ func lspList(apiKeys []string) ([]string, error) {
 	}
 	return lspList, nil
 }
+
+func lspFullList(apiKeys []string) ([]string, []string, error) {
+	type void struct{}
+	var member void
+
+	rows, err := pgxPool.Query(context.Background(),
+		`SELECT
+			COALESCE(lsp_ids->'active', lsp_ids) as active_lsp_ids,
```

Comment:
I understood that as a backward computability attempt without the need of migration.
E.g if active exists (new form of data) then use it, otherwise return lsp_ids (old form of data).
@yaslama can you elaborate?

Reply:
The field can contain an array of ids or an object: ```{"active":activeArray, "inactive":inactiveArray]```. If it's an array and not an object, it is considered as the content of ```activeArray```. Exactly what @roeierez wrote

Reply:
I see, makes sense